src/main/resources/templates/db_scripts/schema.json

```diff
@@ -19,29 +19,79 @@
   "tables": [
     {
       "tableName": "alert",
-      "generateId": true,
```

Comment:
please upgrade the pom.xml to use latest RMB version for this change to take effect